docs/UserGuide.md

```diff
@@ -380,11 +380,23 @@ Redo the most recent command that was undone.
 
 Format: `redo`
 
-### Clearing all entries : `clear`
+<div style="page-break-after: always;"></div>
+
+### Viewing help : `help`
 
-Clears all entries from the internship tracker.
+Display the command summary and a message explaining how to access the help page.
```

Comment:
I think this sounds more user-centric: 
"If you need a quick refresher on the available commands, head right to the help page. Link to the user guide is also available here."

Reply:
updated